src/limiter.rs

```diff
@@ -56,6 +56,13 @@ impl<I> Bucket<I> {
         }
     }
 
+    /// Revert the previous consumption of the given number of bytes.
+    /// 
+    /// This method should only be called when the speed is finite
+    fn revert_consume(&mut self, size: f64) {
```

Comment:
ask for a better naming

Reply:
there was `getc`/`ungetc` so i'd say `unconsume` 🙃